apps/web/prisma/client/schema.prisma

```diff
@@ -5,14 +5,71 @@ generator client {
   provider = "prisma-client-js"
   output = "./client"
 }
-
+// TODO: Change ID to UUID
 datasource db {
   provider = "postgresql"
   url      = env("DATABASE_URL")
 }
 
 model Subscribers {
-  id          BigInt @id @default(autoincrement())
-  created_at  DateTime @default(now())
-  email       String @unique
+  id                String            @id      @default(uuid())
+  Email             String            @unique
+  Name              String
+  LinkedInUrl       String
+  GitHub            String?
+  YearsOfExperience Int
+  Skills            Json?
```

Comment:
Is there a way to type the Json of Skills?

Reply:
Nope, Json is a generic [type](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-json-fields), but maybe we could create another model called Skills related to the Subscribers models.

Reply:
I'm gonna mark it as resolved and we discuss it later